From fe23b2dc1e2c19c1d1d5e3efb2929e488062e347 Mon Sep 17 00:00:00 2001
From: Giuseppe Maggio <jertlokk@gmail.com>
Date: Wed, 2 Jan 2019 12:11:38 +0000
Subject: [PATCH] angler: QCamera: build with -Wno-unused-parameter

Let's just avoid to clutter up the whole source code and silence such
warning (for now, or forever *shrugs*).

Change-Id: I341e35251382dbcb04d2716c3718cd71734288bf
---
 camera/QCamera2/Android.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/camera/QCamera2/Android.mk b/camera/QCamera2/Android.mk
index f777763..f104fb5 100644
--- a/camera/QCamera2/Android.mk
+++ b/camera/QCamera2/Android.mk
@@ -38,7 +38,7 @@ LOCAL_SRC_FILES += \
         HAL/QCameraParameters.cpp \
         HAL/QCameraThermalAdapter.cpp
 
-LOCAL_CFLAGS := -Wall -Wextra -Werror -Wno-unused-label
+LOCAL_CFLAGS := -Wall -Wextra -Werror -Wno-unused-label -Wno-unused-parameter
 LOCAL_CFLAGS += -DHAS_MULTIMEDIA_HINTS
 LOCAL_CFLAGS += -DENABLE_MODEL_INFO_EXIF
 
